README.md

```diff
@@ -0,0 +1,99 @@
+<div align='center'>
+<br />
+<img src='./logo/theia.svg' alt='theia logo' width='125'>
```

Comment:
Sorry, should have noticed this earlier: this logo doesn't exist and would probably be inappropriate (should be a CDT.cloud logo if any).

I'd suggest just eliminating the HTML `<div>` altogether and going with a markdown `# ECLIPSE CDT.CLOUD - CLANGD CONTEXTS` header instead.

Reply:
Right, good idea! I force-pushed a new change.